ui/components/multichain/pages/send/components/quote-card/hooks/useEthFeeData.tsx

```diff
@@ -32,14 +34,15 @@ export default function useEthFeeData(gasLimit = 0) {
   const networkAndAccountSupports1559 = useSelector(
     checkNetworkAndAccountSupports1559,
   );
-  const { medium, gasPrice: maybeGasFee } = useSelector(getGasFeeEstimates);
-
-  // remove this logic once getGasFeeEstimates is typed
-  const gasFee1559 = maybeGasFee ?? medium?.suggestedMaxFeePerGas;
+  // Need to use estimates from gas fee controller instead of transaction gas estimates bc transaction gasEstimates don't get cleared after submitting a POL tx
+  const { medium } = useSelector(getGasFeeControllerEstimates);
```

Comment:
This is the only change that you need. If the `confirmTransaction` state exists, the `getGasFeeEstimates` takes the `maxFeePerGas` and `maxPriorityFeePerGas` from it. Otherwise it gets those values from the gas fee controller state directly.

This should be a temporary fix, because I believe that the main issue is that we are not clearing the confirmation redux state after successfully submitting the confirmation successfully (`store.confirmTransaction`). FYI @matthewwalsh0